File: frontend/henn-laszlo/projects/content-data-access/src/lib/models/update-article-request.ts
export interface UpdateArticleRequest {
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly summaryHu: string | null;
  readonly summaryEn: string | null;
  readonly contentHu: string;
  readonly contentEn: string | null;
  readonly displayOrder: number;
}

File: backend/src/Modules/Modules.Content.Presentation/Articles/Update/UpdateArticleRequest.cs
namespace Modules.Content.Presentation.Articles.Update;

internal sealed record UpdateArticleRequest(
    string TitleHu,
    string? TitleEn,
    string? SummaryHu,
    string? SummaryEn,
    string ContentHu,
    string? ContentEn,
    int DisplayOrder);